modules/auxiliary/scanner/http/apache_webserver_traversal.rb

```diff
@@ -0,0 +1,71 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Auxiliary::Report
+  include Msf::Auxiliary::Scanner
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Path Traversal in Apache 2.4.49',
+        'Description' => %q{
+          This module exploits an unauthenticated directory traversal vulnerability which exists in Apache version 2.4.49.
+          If files outside of the document root are not protected by ‘require all denied’ these requests can succeed.
+        },
+        'Author' => [
+          'Ash Daulton', # Vulnerability discovery
+          'Dhiraj Mishra', # Metasploit Module
+        ],
+        'License' => MSF_LICENSE,
+        'References' => [
+          ['CVE', '2021-41773']
+        ],
+        'DisclosureDate' => '2021-10-05',
+        'Platform' => 'ruby',
```

Comment:
I don't think this is necessary to define. This should be the target platform, but this module isn't delivering a payload.